src/pybrake/metrics.py

```diff
@@ -136,3 +137,57 @@ def _resume(self):
         if not self._paused():
             return
         self.start_time = pytime.time()
+
+
+def send(url, headers, failed_stats, payload=None, method=None):
```

Comment:
Can you please perform this refactoring in a separate PR? We will merge it prior to merging this one. This looks unrelated to adding the backlog aka failed stats.

Reply:
Sure